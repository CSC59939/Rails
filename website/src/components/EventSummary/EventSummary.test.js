import React from 'react';
import renderer from 'react-test-renderer';
import { shallow, mount, render, configure} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { Button, Card, Icon } from 'antd';
import { EventSummary } from '..';
import '../../utils/tests/test.css';

configure({ adapter: new Adapter() });

describe("EventSummary", () => {
  it('Render without any crash, render the EventSummary', () => { // eslint-disable-line no-undef
    expect(renderer.create(<EventSummary course="Cs103" eventName="Exam" dueDate="Tomorrow" />).toJSON()).toMatchSnapshot();
  });
})
