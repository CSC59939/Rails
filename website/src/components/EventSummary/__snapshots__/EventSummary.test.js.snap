// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EventSummary Render without any crash, render the EventSummary 1`] = `
<div
  className="EventSummary"
  style={
    Object {
      "borderColor": "red",
      "color": "red",
    }
  }
>
  <span
    className="course"
  >
    Cs103
  </span>
  <span
    className="eventName"
  >
    Exam
  </span>
  <span
    className="dueDate"
  >
    Tomorrow
  </span>
</div>
`;
